packages/mobile/locales/en-US/walletFlow5.json

```diff
@@ -72,14 +75,28 @@
   "feedItemVerificationRewardInfo": "Verifier Reward",
   "feedItemFaucetTitle": "Celo",
   "feedItemFaucetInfo": "{{faucet}} Faucet",
-  "feedItemFaucetInfo_missingTestnet": "Faucet",
+  "feedItemFaucetInfo_noTestnet": "Faucet",
   "feedItemInviteSentTitle": "Celo",
   "feedItemInviteSentInfo": "Invited {{nameOrNumber}}",
-  "feedItemInviteSentInfo_missingInviteeDetails": "$t(inviteFlow11:inviteSent)",
+  "feedItemInviteSentInfo_noInviteeDetails": "$t(inviteFlow11:inviteSent)",
```

Comment:
oh what is happening with the value here? is this some i18n magic?

Reply:
It's the [`context` feature of `i18next`](https://www.i18next.com/translation-function/context), it's appended to the key.

Reply:
😲